code/modules/food_and_drinks/kitchen_machinery/processor.dm

```diff
@@ -202,6 +220,15 @@
 	if(src.processing)
 		user << "<span class='warning'>The processor is in the process of processing!</span>"
 		return 1
+	if(user.a_intent == "grab" && user.pulling && isliving(user.pulling) && isslime(user.pulling) || ismonkey(user.pulling))
```

Comment:
Somewhere along the line being able to manually put in mobs was broken. So I basically took this code from tables.
If anyone wants me to put in a delay just mention it.